bitnami/memcached/Chart.yaml

```
@@ -31,4 +31,4 @@ maintainers:
 name: memcached
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/memcached
-version: 7.4.14
+version: 7.4.15
```

```suggestion
version: 7.5.0
```

As you are including new parameters, you need to bump the version to a new minor one